dev-manuals/architecture/template.md

```diff
@@ -0,0 +1,38 @@
+# Architecture of `$SERVICE`
+
+## Purpose
+
+`$PURPOSE`, i.e.
+```md
+Being a fun-to-play minigame that lets users choose one correct answer from a bunch of wrong answers
+```
+
+## General code structure
+
+`$STRUCTURE`,
+i.e.
+```md
+This backend has the following package structure
+- `data`: Contains the data objects and DTOs
+- `controllers`: Containers the server routes
+- `services`: Contains the business logic
+```
+
+## Starting the service
+
+See the README or the user manual.
+
+## Known Design Flaws
+
+- `$LIST OF DESIGN FLAWS PRESENT AT THE MOMENT`
```

Comment:
what do you mean with design flaws?

Reply:
If we know of any systemic flaw that simply won't be solvable without great refactors, i.e. `having multiple correct answers for chickenshock`.
It is of course always nice if we can leave this section empty…